pyproject.toml

```diff
@@ -68,6 +68,7 @@ classifiers = [
 dependencies = [
     "transformers>=4.57.1",
     "datasets",
+    "accelerate",
```

Comment:
n00b question: why is accelerate needed? we'd removed this in https://github.com/NVIDIA-NeMo/Megatron-Bridge/pull/798 because there were no direct usages, so i want to double check how it's used here

Reply:
device_map="cuda" from HF now requires accelerate to be installed. I think it related to HF's device management.